02-peer-protocol.md

```diff
@@ -207,12 +207,16 @@ definitions they reuse even feature bits, but they are not an
 arbitrary combination (they represent the persistent features which
 affect the channel operation).
 
-The currently defined types are:
+The currently defined basic types are:
   - no features (no bits set)
   - `option_static_remotekey` (bit 12)
   - `option_anchor_outputs` and `option_static_remotekey` (bits 20 and 12)
   - `option_anchors_zero_fee_htlc_tx` and `option_static_remotekey` (bits 22 and 12)
 
+Each basic type has the following variations allowed:
```

Comment:
are these mutually exclusive

Reply:
I assume you're saying we should be explicit in the text to say that they are not mutually exclusive? You obviously want to support both.